Hooks.php

```diff
@@ -62,6 +68,15 @@ public static function onParserFirstCallInit(Parser &$parser) {
     }
   }
 
+ /*   public static function onParserFirstCallInit(Parser &$parser) {
+        global $wgGoogleRichCardsAnnotateBooks;
+        
+        if($wgGoogleRichCardsAnnotateBooks) {
+            $book = Event::getInstance();
```

Comment:
`Book::getInstance();`

Reply:
I'm not sure of getting it: if I use 
            $book = Book::getInstance();
it crashes

Reply:
Check out logs of your HTTP server, the reason should be there

Reply:
@giby probably right after HTTP request to mediawiki pages it should appear in error log(usually HTTP servers, both of nginx and apache/httpd writes two log files: access.log and error.log). Your error/exception traceback should be there.

Reply:
So my hosting provider proposes me
2018-11 : web - ftp - error  - out 
I cheacked "web" and "error" and nothing related to mediawiki…

Reply:
@giby probably your provider doesn't have logs verbose enough for that, give me some time to check it out.

Reply:
I made crach my wiki, and get a 504 error :(

Reply:
@giby So simply restore the extension :) I hope you used it right from git, so you could easily switch between exact commits.

Btw, for development I would like to suggest you to use Docker, which will bring you fast local development workflow no matter if you are using macOS or Windows. To deploy wiki+database for development it will take about 1 minute with that workflow.

PS. Development right on running project(i.e. production) is not such a good idea ;)